graphene/types/objecttype.py

```diff
@@ -20,12 +20,14 @@ class ObjectTypeOptions(BaseOptions):
 
 
 class ObjectTypeMeta(BaseTypeMeta):
-    def __new__(cls, name, bases, namespace):
+    def __new__(cls, name_, bases, namespace, **kwargs):
```

Comment:
We shouldn't use `**kwargs` here. It's intended that the `ObjectTypeMeta` (and any other `*Meta`) have explicit defined values and types.